ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -28,14 +28,14 @@
       {% endif %}
     cifmw_edpm_deploy_baremetal_operators_build_output: "{{ operators_build_output }}"
 
-- name: Install OpenStack operator
+- name: Install Dataplane with baremetal
   vars:
     make_edpm_compute_baremetal_env: "{{ cifmw_edpm_deploy_baremetal_common_env |
```

Comment:
Since we are changing the target name here. We also need to change the var names associated with `make_edpm_baremetal` target.
Here is the generated [taskfile](https://logserver.rdoproject.org/39/339/1c9ae0dbd553a046a537672445a9725bff4c90a5/github-check/ci-framework-crc-podified-edpm-baremetal/3385591/ci-framework-data/artifacts/roles/install_yamls_makes/tasks/make_edpm_baremetal.yml) for `make_edpm_baremetal`

We need to change make_edpm_compute_baremetal_env to make_edpm_baremetal_env and same for dry run var.

Reply:
ack, changed.. But seems to have  not created any issue without those vars but failed attaching the interface as we did run the attach_default_interface make target in edpm_prepare. 

https://logserver.rdoproject.org/39/339/a173897b0f87282b7c5ddcd6c5bbffbd8abc961b/github-check/ci-framework-crc-podified-edpm-baremetal/5af6a1a/ci-framework-data/logs/ci_make_004_run_edpm_baremetal.log

Reply:
Yes, `make crc_attach_default_interface` is enabled by default in edpm-prepare. Thank you for disabling it.

Reply:
New vars are applied here defined under[ BM scenario file]( https://github.com/openstack-k8s-operators/ci-framework/blob/main/scenarios/centos-9/edpm_baremetal_deployment_ci.yml#L4) : https://logserver.rdoproject.org/39/339/b5bdd79e0a50731af3b14a4af5b4358bbac96ef2/github-check/ci-framework-crc-podified-edpm-baremetal/fb1c415/ci-framework-data/logs/ci_make_003_run_edpm_baremetal.log